python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2896,15 +2903,29 @@ def _parse_import_prerequisites(self, graph):
         """
         missing_operators = set()
         for node in graph.node:
+            try:
+                from tensorflow.python.framework import op_def_registry
+            except ImportError as e:
+                raise ImportError(
+                    "Unable to import tensorflow which is required {}".format(e))
+            getOpDef = op_def_registry._registered_ops.get if hasattr(op_def_registry,\
+                        "_registered_ops") else op_def_registry.get
+            op_def = getOpDef(node.op)
             if node.op == "Placeholder" or node.op == 'PlaceholderWithDefault':
                 pass
             elif node.op == "Const":
                 pass
+            elif node.op in ["PartitionedCall", "StatefulPartitionedCall"]:
+                pass
             else:
                 if any([node.op in t for t in [_identity_list, _convert_map,
                                                _convert_map_rnn,
                                                _control_flow_nodes]]):
                     pass
+                elif op_def is not None and op_def.is_stateful:
+                    raise Exception("Found a stateful operator in this graph {}. "\
```

Comment:
Better to add this to missing op list (with extended info if needed) still instead of exception as we may miss over all missing ops list.

Reply:
@srkreddy1238 , Sure that's a good idea. Then i can create a list of stateful_partitioned_ops(eg. StatefulPartitioned,PartitionedOp) just like _freezed_graph_pruned_op_list to provide extended info to the user. Does it make sense?

Reply:
Objective of this function is to give a consolidated list of unsupported ops.
Any approach is good as long as the purpose is met, which is showing all unsupported ops instead of just an exception.

Reply:
Yes sure, It will support the aforesaid purpose. You can review once after I handle this in the new implementation.

Reply:
Completed this. Please review once.